agm_minutes/2015AGM_minutes.md

```diff
@@ -0,0 +1,115 @@
+
+Since John's death, Zeth has been Acting Chair.
+Zeth said that because of these changes, we did not change the venue this year.
+
+Zeth said that does not want to be chair again, that in previous years John has shouldered the burden, and that he does not want anybody to burn out.
+
+Zeth said that constitutionally, we just need a committee.
+The committee will meet at a "wash up" event in the autumn, where who does what can be decided.
+Travel costs for the meeting will be covered by the Society.
+
+As such, Zeth said that he would like to focus on a bid to bring EuroPython to the UK.
+He then proposed that Peter be "General Secretary", to be "responsible for crew and communications", and that Daniele Procida be "Local Chair", responsible for arrangements with the venue.
+
+Zeth then discussed how the conference has been planned for this year, including mentioning the use of GitHub issues.
+He also mentioned that the conference website is built using Python 2.
+Russel Winder has volunteered to update the conference website to use Python 3.
+
+Zeth then thanked Nick, Peter, Mary, Sarah, Luke, Naomi, and others who had worked to put the conference on, and asked for a standing ovation for Nick, who is standing down from the committee this year and who has been responsible for the success of the education track.
+
+Zeth said the conference was planned, the conference was organised, and the conference was done.
+
+## Treasurer's report
+
+Mary presented the income and expenditure accounts for the 2014 conference, which can be found at https://github.com/PyconUK/PyConUKSociety/blob/master/agm_finances/2014AGM_finances.md
+
+For 2014 we made a profit of around £5000.
+
+Mary also reported that around £1700 has been raised for Cancer Research so far at the conference, and that there's a bucket at reception that people can use for further donations.
+
+At this point, Nick thanked Mary and asked for a standing ovation.
+
+## Proposal for financial support plan
+
+Zeth asked Nick to present a new financial support plan.
+
+Nick proposed that we use some of the profits from this year's conference to provide financial support to delegates for next year.
+
+This was voted for by a show of hands.
+
+## Statement on diversity
+
+When people bought tickets, they were asked for their gender.
+Zeth showed a graph of the responses, which showed that 20% of delegates identify as female, 72% of delegates identify as male, 2% as non-binary, with the remainder not answering the question.
+
+Zeth also said that we had had many more applications to DjangoGirls than could be accepted.
+
+## Statement on education
+
+Nicholas gave a brief account of the education track and said that he felt it had been successful.
+
+## Statement on co-operation with the PSF
+
+Zeth said that because of the size of conference budget, the PyCon UK Society will need to become a charity, and that we will work with Van and the PSF to do this.
```

Comment:
How about if Sarah changes the line to the following more vague statement?:

Zeth said we will reconsider our organisational status and Van and the PSF
will be providing help and assistance to do this.

On 24 September 2015 at 06:28, Mary Mooney <notifications@github.com> wrote:

> In agm_minutes/2015AGM_minutes.md
> <https://github.com/PyconUK/PyConUKSociety/pull/10#discussion_r40286863>:
>
> > +This was voted for by a show of hands.
> > +
> > +## Statement on diversity
> > +
> > +When people bought tickets, they were asked for their gender.
> > +Zeth showed a graph of the responses, which showed that 20% of delegates identify as female, 72% of delegates identify as male, 2% as non-binary, with the remainder not answering the question.
> > +
> > +Zeth also said that we had had many more applications to DjangoGirls than could be accepted.
> > +
> > +## Statement on education
> > +
> > +Nicholas gave a brief account of the education track and said that he felt it had been successful.
> > +
> > +## Statement on co-operation with the PSF
> > +
> > +Zeth said that because of the size of conference budget, the PyCon UK Society will need to become a charity, and that we will work with Van and the PSF to do this.
>
> [not for the minutes] but we need to consider other options other than
> Charity. There is CIC, Limited By Guarantee and Cooperative. Charity may be
> too restrictive.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PyconUK/PyConUKSociety/pull/10/files#r40286863>.
>